### ISSUE TITLE
Feature Request : Equivalent Georgian/Islamic date below Shamsi

### ISSUE THREAD

**Issue by sirpooya** [moallemi/persian-calendar-mac-issues#6]
An Option to add Georgian month & day beside Shamsi month/day in Menu bar.

**Comment by sirpooya** [comment]
Dear @moallemi this is what I'm talking about:
<img width="329" alt="Screen Shot 1399-04-06 at 3 42 18 PM" src="https://user-images.githubusercontent.com/1027824/85851622-507c0700-b7c4-11ea-9413-ab7c99e7214d.png">

It's much easier to lookup equivalent date in Gregorian & Islamic in a single line below Shamsi date than to look it up in calendar table.

**Comment by sirpooya** [comment]
Dear @moallemi this is what I'm talking about:

It's much easier to lookup equivalent date in Gregorian & Islamic in single line below Shamsi than to look it up in calendar table.

**Comment by moallemi** [comment]
By `Single line below Shamsi date` you mean displaying them in menu bar?

**Comment by sirpooya** [comment]
Exactly, Gregorian date can be shown in the menu bar beside Shamsi date (with a checkbox to turn it off/on).
But more to this feature is to show both Gregorian & Islamic in the popover window below Shamsi date, pretty much like in the photo I attached.

**Comment by moallemi** [comment]
Thanks for your feedback. I will add an option to display Gregorian and Hijri dates in menu bar and popover in next update.
